idl/github.com/uber/cadence/history.thrift

```diff
@@ -343,6 +343,12 @@ struct QueryWorkflowResponse {
   10: optional binary queryResult
 }
 
+struct ReapplyEventsRequest {
+  10: optional string domainID
+  20: optional shared.WorkflowExecution workflowExecution
+  30: optional list<shared.HistoryEvent> events
```

Comment:
should be DataBlob

Reply:
the deserialization of DataBlob should happen in history handler layer, meaning history engine should see the deserialized history events

Reply:
done